### PR TITLE
fakeroot: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by dawidd6**
This is an automated pull request to build a new bottle for linuxbrew-core
based on the existing bottle block from homebrew-core.

### CODE REVIEW

**Comment by issyl0** [comment]
I tried to fix the build in #14543 and it was harder than I initially expected. Please do try again! It will fix an open issue. 🙏

**Comment by dawidd6** [comment]
Seems like I did it, yay!
This time I was so thrilled, I even watched the live output on azure.

File: Formula/fakeroot.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ class Fakeroot < Formula
     sha256 "4e0afd9c97c78fa0bf2c16420d21d26e4320a0a3eac94ba381b132f701ea24dc" => :el_capitan
   end
 
+  unless OS.mac?
+    depends_on "libtool" => :build
+    uses_from_macos "libcap"
```
Comment (issyl0) [inline Formula/fakeroot.rb]: This doesn't need to be in an `if OS.mac?` block and can even be pushed upstream.

Comment (dawidd6) [reply Formula/fakeroot.rb]: `libtool` too?

Comment (issyl0) [reply Formula/fakeroot.rb]: Oh, sorry, I was referring only to line 17.

Comment (dawidd6) [reply Formula/fakeroot.rb]: Actually `libtool` might not even be needed.

**Pull request closed without merging**